$OpenBSD: patch-bin_activemq,v 1.1.1.1 2013/02/05 15:08:45 jasper Exp $

Don't reserve more memory upfront than the default limits allow.

--- bin/activemq.orig	Tue Oct  2 18:15:33 2012
+++ bin/activemq	Tue Feb  5 08:43:45 2013
@@ -151,9 +151,9 @@ fi
 JAVACMD="auto"
 
 # Set jvm memory configuration
-if [ -z "$ACTIVEMQ_OPTS_MEMORY" ] ; then
-    ACTIVEMQ_OPTS_MEMORY="-Xms1G -Xmx1G"
-fi
+#if [ -z "$ACTIVEMQ_OPTS_MEMORY" ] ; then
+#    ACTIVEMQ_OPTS_MEMORY="-Xms1G -Xmx1G"
+#fi
 
 # Uncomment to enable audit logging
 #ACTIVEMQ_OPTS="$ACTIVEMQ_OPTS -Dorg.apache.activemq.audit=true"
